diff -urN aaa/fung-calc-1.3.2b/src/fung-calc.desktop fung-calc-1.3.2b/src/fung-calc.desktop
--- aaa/fung-calc-1.3.2b/src/fung-calc.desktop	2003-07-04 01:22:01.000000000 +0200
+++ fung-calc-1.3.2b/src/fung-calc.desktop	2003-12-12 17:38:31.000000000 +0100
@@ -3,10 +3,11 @@
 Type=Application
 Exec=fung-calc
 Icon=fung_calc
-DocPath=fung-calc/index.html
+X-DocPath=fung-calc/index.html
 Comment=2D and 3D graphing calculator
-Terminal=0
+Terminal=false
 Name=Fung-Calc
 MimeType=application/x-fgc
 X-KDE-NativeMimeType=application/x-fgc
-Categories=Qt;KDE;Mathematics
+Categories=Qt;KDE;Education;Science;Math;
+Encoding=UTF-8
